### ISSUE TITLE
Resend Email Confirmation Link

### ISSUE THREAD

**Issue by kristynfudge** [librecores/librecores-web#460]
Link takes you to a page with this information:
We could not find what you were looking for
The requested resource couldn't be located.
Checkout for any URL misspelling or an outdated bookmark. If an internal link led you here, please let us know.

**Comment by agathver** [comment]
It was a 404. Can you describe on which page did you get the link that led to this 404 page?

**Comment by kristynfudge** [comment]
Yeah. I was creating an account and that brought me the page that states an email will be sent to confirm my account. On that page it states if you don't receive an email click here to have it resent. Clicking the `here` hyperlinked text took me to the 404. I also never received an email to confirm my account and I can't login until I do, so I'd love any help with that if can assist.

**Comment by agathver** [comment]
@imphil Can you check, twig code looks fine to me I see no reason why anyone should get 404
